test/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/generators/Acc14Gen.scala

```diff
@@ -29,7 +29,7 @@ import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.generators.PhoneNumberGe
 
 object Acc14Gen {
 
-  def genNdrcDetails: Gen[NdrcDetails] = for {
+  val genNdrcDetails: Gen[NdrcDetails] = for {
```

Comment:
There will be a conflicts - I used `lazy val` which might benefit if you run only single test

Reply:
rebased